src/Magento/FunctionalTestingFramework/DataTransport/Auth/WebApiAuth.php

```diff
@@ -63,7 +70,13 @@ public static function getAdminToken($username = null, $password = null)
         }
 
         if (isset(self::$adminAuthTokens[$login])) {
-            return self::$adminAuthTokens[$login];
+            $tokenLifetime = getenv('MAGENTO_ADMIN_WEBAPI_TOKEN_LIFETIME');
```

Comment:
Do you need a default value in case user does not set MAGENTO_ADMIN_WEBAPI_TOKEN_LIFETIME?

Reply:
no it'd be false.  $isTokenExpired checks truthy value of $tokenLifetime on next code line and short circuits to false

Reply:
Do you want a default value?

Reply:
I see. No need default value.